### ISSUE TITLE
[Suggestion] mob blacklist for certain settings

### ISSUE THREAD

**Issue by DeshiDesu** [Corosauce/mobtimizations#4]
Im using the iceandfire mod and the dragons are static once they leave the player proximity.

The dragons are meant to have longer render distance than most common mobs e.g pigs and cows as they are meant to be seen from far away.

The problem is I do not want to adjust the rangecutoff setting since that would obviously decrease performance.

I would appreciate it if you would add a whitelist/blacklist for certain mobs so they wouldn't be affected by these changes.

1.20.1
Forge 47.3.1
mobtimizations-forge-1.20.1-1.0.0.jar

**Comment by Texbio** [comment]
It would also be nice if we could nametag the mob with a string to prevent it from being affected, also in a pack I am playing, I am using armor stands for a redstone contraption, would be nice if those were not affected

**Comment by SparklingEagle** [comment]
Yes, I am using mods that add mortars and when I shoot them the shells just stay floating in the sky forever. I wish I could allow them to act freely. Could you help us, @Corosauce?
